examples/tictactoe/tictactoe.py

```diff
@@ -51,9 +51,7 @@ def play(self, move: Move) -> None:
             assert Txn.sender == self.host, "It is the host's turn"
             player = arc4.UInt8(HOST)
         else:
-            assert Txn.sender == self.challenger.get(
-                Global.zero_address
-            ), "It is the challenger's turn"
+            assert Txn.sender == self.challenger.get(Account()), "It is the challenger's turn"
```

Comment:
This reads confusingly, like what is `Account` in this case. I can't look at this an intuitively know. Why is anything needed in `get` anyway?

Reply:
The parameter is the default value if the state var is not set, but I agree that `Account()` is confusing

Reply:
Could be simplified by making `self.challenger` use Global.zero_address as the unset/initial value?

Reply:
so `.get()` in Python mappings would return `None` if the key isn't present, that's the same idea here with `GlobalState` proxies. I'll add `default=` in this example, but using `GlobalState` instead here vs initialising to zero like you've suggested @daniel-makerx was what the contract initially did, the way it is not generates more efficient code so is useful to point out in an example.

To someone fluent in Python `.get()` and the default arg makes total sense and this reads well to me at least, although if AVM had enough support that we could utilise `None` it'd be even better.